pallets/pool-xyk/src/mock.rs

```diff
@@ -195,6 +195,7 @@ parameter_types! {
     pub const GetBuyBackDexId: DEXId = 0;
 }
 
+// TODO: Not mocked, because of different DEX types
```

Comment:
As I got, the problem is that `DEXId` hardcoded to only 2 values and the tests require custom DEXId. We could introduce `pub type MockDexId = u32;` in `common/mocks` for such cases.

Reply:
The same for `pallets/liquidity-proxy/src/mock.rs`.

Reply:
Sorry, I don't understand, now it is already used as `u32` for the cases, why do we need `type MockDexId`? If we are not going to change all `DEXId` to it, it may be little bit confusing. Actually I just realized that we may:
1) remove the parameter `GetBuyBackDexId` and set up it manually;
2) set up type as macro parameter;
3) leave `GetBuyBackDexId`, but in a such way `pub GetBuyBackDexId: DEXId = DEXId::from(common::DEXId::Polkaswap);`
To my mind last one is nice enough, isn't it?